site/config.yaml

```diff
@@ -57,8 +57,10 @@ params:
   docs_search_index_name: index_name
   docs_search_api_key: api_key
   docs_versioning: true
-  docs_latest: v0.0.6
+  docs_latest: v0.0.7
   docs_versions:
+    - main
```

Comment:
wdyt of `latest`? I think `main` indeed makes sense from the perspective of "versioning is based on git references", but when you look at the website, I think the concept of "latest version" .. "version 0.0.6 .." makes more sense - wdyt?

Reply:
feedback from community call: 

- `latest` could instead be used as a pointer to the latest released version (analogous to how https://github.com/vmware-tanzu/cartographer/releases/latest points at whatever latest released tag we have)
- maybe call this .. `preview`? maybe `develpment`?

Reply:
My vote is for `development` as it is clear for users that it refers to unreleased software on which the team is actively working. For me as user, `latest` refers to the most recently released and stable version, and `preview` sounds like it will eventually see the light but not sure when :)

Please upvote your options, as soon as we have consensus, I can update the PR

Reply:
@squeedee what did we come up with in the docs meeting?

Reply:
"Edge" IIRC.

Reply:
100% agree "Latest" is latest-known-good.. I'm also ok with "Development"

Reply:
Another favorite from community meetings was Development (putting here so people can vote)

Reply:
Updated PR with "development" for unreleased version docs. Using `latest` along the most recent version number would add at least two additional steps to the process whenever we need to update for a new release